frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ImportCluster/schema.json

```diff
@@ -0,0 +1,147 @@
+[
```

Comment:
Is it required now that the yaml editor requires schema?

Reply:
this is the schema definitions for ImportCluster--what properties ManagedCluster/Secret/KlusterletAddonConfig require